pkg/broker/ingress/handler.go

```diff
@@ -145,10 +146,13 @@ func (h *Handler) ServeHTTP(response nethttp.ResponseWriter, request *nethttp.Re
 	defer span.End()
 	if span.IsRecordingEvents() {
 		span.AddAttributes(
-			kntracing.MessagingSystemAttribute,
-			tracing.PubSubProtocolAttribute,
-			kntracing.BrokerMessagingDestinationAttribute(broker),
-			kntracing.MessagingMessageIDAttribute(event.ID()),
+			append(
```

Comment:
Is it worth it to just call `span.AddAttributes` twice, rather than creating a temporary slice?

Reply:
Two separate calls would result two smaller slices being allocated instead of one larger slice. I also suspect in this case that go will be able to allocate the temporary slice on the stack. Either way, I'm not sure which is more performant and would be surprised if the difference between the two ends up being very significant. Additionally each call to AddAttributes must take a mutex so I slightly prefer keeping just a single call.